CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ if (fletch_BUILD_WITH_CUDA)
     set( CUDNN_ROOT "" CACHE PATH "CUDNN root folder" )
     mark_as_advanced( CUDNN_ROOT )
   endif()
+
+  option(fletch_BUILD_CUDA_CSTD98 "Build all cuda code with C++ standard 98" FALSE)
```

Comment:
Why build the Cuda code differently from other code? Is it a problem to enable?disable C++ 11 for the entire build?